packages/hdwallet-native/src/crypto/classes/cipherString.ts

```diff
@@ -44,4 +45,13 @@ export class CipherString {
   get encryptedString() {
     return `${this.encryptionType}.${[this.data, this.iv, this.mac || ""].join("|")}`;
   }
+
+  toEncryptedObject(key: SymmetricCryptoKey): EncryptedObject {
```

Comment:
This helper makes it easier to actually decrypt a CipherString, and will be used in tests later.